src/binder/inc/coreclrbindercommon.h

```diff
@@ -22,7 +22,7 @@ class CLRPrivBinderCoreCLR;
 class CCoreCLRBinderHelper
 {
 public:
-    static HRESULT Init();
+    static HRESULT Init(bool isBundle);
```

Comment:
Binder cannot access AppDomain members since the type is not constructed yet. The fIsBundle flag is similar to other variables such as `AppBaseURL; DynamicDirectory; DynamicBaseDirectory; AppName; AppConfigFile; AppDomainId`. But fIsBundle may not be necessary if we make BundleInfo a global.

(If we have single-file plugins in future, we may need to come back and think about this interface).